corehq/ex-submodules/soil/progress.py

```diff
@@ -167,7 +167,21 @@ def progress_complete():
 def _is_real_task(task):
     # You can look up a task with a made-up ID and it'll give you a meaningless task object
     # Make sure the task object you have corresponds to an actual celery task
-    return task and task._get_task_meta().get('task_name') is not None
+    if task:
+        # Non-real "tasks" will have all null values except for
+        #   - status: "PENDING"
+        #   - task_id: <task_id>
+        # If ANYTHING else is set, we give it the benefit of the doubt and call it real
+        return any(
+            value
```

Comment:
Would it make sense to make this more strict? What if value is `0` or `{}` etc.? Are those considered "null"?
```suggestion
            value is not None
```

Reply:
Yup, thanks I think that's a good change